packages/cudos-utilities/get-node-status.js

```diff
@@ -28,35 +28,33 @@ async function getStatusNodeByUrl(url) {
     nodeStatus.statusCode = response.status
 
     if (nodeStatus.statusCode === 200) {
-      if (typeof response.data.result.node_info === 'undefined') {
-        nodeStatus.isConnected = false
-        nodeStatus.errorMessage = 'Missing data from node response. Are you sure you connected a node?'
+      if (response.data.result && response.data.result.node_info) {
+        // node is up and running
+        nodeStatus.isConnected = true
+        nodeStatus = attachAddidionalInfo(nodeStatus, response.data.result)
         return nodeStatus
       }
-      // node is up and running
-      nodeStatus.isConnected = true
-      nodeStatus = attachAddidionalInfo(nodeStatus, response.data.result)
+      // probably connected to something other than node
+      nodeStatus.isConnected = false
+      nodeStatus.message = 'Missing data from node response. Are you sure you connected a node?'
       return nodeStatus
     }
     // status is not 200
     nodeStatus.isConnected = false
+    nodeStatus.message = 'Unusual node response. Status code should be 200.'
     return nodeStatus
   } catch (ex) {
-    nodeStatus.isConnected = false
-    if (typeof ex.code !== 'undefined') {
-      nodeStatus.statusCode = ex.code
-      return nodeStatus
+    if (!ex.isAxiosError) {
```

Comment:
didn't we come to an agreement that we would. use `error` isntead of exception

Reply:
I missed that, fixing it right away